packages/edit-post/src/store/selectors.js

```diff
@@ -379,7 +379,6 @@ export const isMetaBoxLocationVisible = createRegistrySelector(
 			isMetaBoxLocationActive( state, location ) &&
 			getMetaBoxesPerLocation( state, location )?.some( ( { id } ) => {
 				return select( editorStore ).isEditorPanelEnabled(
-					state,
```

Comment:
This was making `isMetaBoxLocationVisible` unusable. Seemed a simple enough fix to just include it here instead of making a separate PR.